internal/server/middleware.go

```diff
@@ -75,7 +75,13 @@ func handleInfraDestinationHeader(c *gin.Context) error {
 // gin.Context.
 // See validateRequestOrganization for a related function used for unauthenticated
 // routes.
-func authenticateRequest(c *gin.Context, tx *data.Transaction, srv *Server) error {
+func authenticateRequest(c *gin.Context, srv *Server) error {
+	tx, err := srv.db.Begin(c.Request.Context())
```

Comment:
this will still work without explicitly starting a transaction, right?

Reply:
Ah yes, I guess that is true. I haven't looked at all the queries, but it might be ok without an explicit transaction.

Reply:
Do you think there is any advantage to not using a transaction? I thought generally we would want to put things into short lived transactions.